src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -31,12 +29,12 @@ public class NpgsqlMigrationsSqlGenerator : MigrationsSqlGenerator
 
         public NpgsqlMigrationsSqlGenerator(
             [NotNull] MigrationsSqlGeneratorDependencies dependencies,
+            [NotNull] IMigrationsAnnotationProvider migrationsAnnotations,
             [NotNull] INpgsqlOptions npgsqlOptions)
```

Comment:
Does EF use C# null reference types or not?

Reply:
Unfortunately not yet - I actually started with some annotation work but it was far too much work for 3.0 so we backed it out. I think it makes sense to wait for EF to get annotated before we do anything here...